index.js

```diff
@@ -9,14 +9,16 @@ import normalizePath from 'normalize-path'
  *   If true, use an absolute path. By default a relative path is used.
  * @property {string} [name='filepath']
  *   The name to export the file path as.
+ * @property {string} [cwd]
+ *   The current working directory to use when generating a relative file path.
```

Comment:
Nit: Can you put this property before `name`? That would make them alphabetized.